.github/workflows/ci.yml

```diff
@@ -40,4 +40,6 @@ jobs:
     - name: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }}
       run: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }}
     - name: pod lib lint
-      run: scripts/podliblint.sh
\ No newline at end of file
+      run: scripts/podliblint.sh
```

Comment:
this is partially my fault but can we make this a separate job so we only run it once per PR rather than 4 times for different build flavor?

Reply:
I can look into it.

Reply:
![Screen Shot 2021-03-25 at 12 58 12 PM](https://user-images.githubusercontent.com/63682282/112535665-ca4c1b80-8d69-11eb-9062-7536d3cf7346.png)

I have updated this.